plugins/package-managers/gradle-inspector/README.md

```diff
@@ -1,17 +1,10 @@
 # GradleInspector
 
-The [GradleInspector] is an alternative analyzer for projects that use the Gradle package manager.
-It is supposed to address [several] [shortcomings] of the "legacy" [Gradle] analyzer, but to not interfere with it, the [GradleInspector] is disabled by default.
+The [GradleInspector] is the new analyzer for projects that use the Gradle package manager.
+It is supposed to address [several] [shortcomings] of the "legacy" [Gradle] analyzer, which is disabled by default now.
```

Comment:
Would it make sense to avoid "the new" and "now" as these terms become outdated?
Maybe instead "the default" and "which is disabled by default"?